pieforproviders/pieforproviders
# 1038/add family fee to api

# 💅🏼 What issue does this fix?
#1036 and #1038

## 🛷 Deployment


## 🧵 Steps to set up locally

`bundle exec rails db:migrate`

## 🧳 Steps to test
- try to process a spreadsheet in the old format (with Transportation revenue) - it should work without throwing errors
- try to process a spreadsheet in the new format (with Family Fee) - it should work without throwing errors
- the front end should load correctly without throwing errors
- when you hit the api for /case_list_for_dashboard transportation fee should be gone and family fee should be present
